src/accelerate/test_utils/testing.py

```diff
@@ -399,6 +399,16 @@ def require_multi_xpu(test_case):
     return unittest.skipUnless(torch.xpu.device_count() > 1, "test requires multiple XPUs")(test_case)
 
 
+def require_multi_gpu_or_xpu(test_case):
+    """
+    Decorator marking a test that requires a multi-GPU setup. These tests are skipped on a machine without multiple
+    GPUs or XPUs.
+    """
+    cuda_condition = is_cuda_available()
+    xpu_condition = is_xpu_available()
+    return unittest.skipUnless((cuda_condition or xpu_condition) and device_count > 1, "test requires multiple GPUs or XPUs")(test_case)
```

Comment:
```suggestion
    return unittest.skipUnless((is_cuda_available() or is_xpu_available()) and device_count > 1, "test requires multiple GPUs or XPUs")(test_case)
```
I don't really see the need for having those other two lines

Reply:
cool